t/rest_server/spec.yaml

```diff
@@ -14,11 +14,43 @@
 
 apiVersion: 'cli.abcxyz.dev/v1alpha1'
 kind: 'Template'
-
-desc:
-  'An example template for a simple HTTP/JSON REST server.'
+desc: 'An example template for a simple HTTP/JSON REST server.'
```

Comment:
Can we remove "example" from this? To me, that makes it sound like it's for people learning about templates rather than being a real template that we expect people to use.

Reply:
Removed "example".